tasks/eth/020-set-gas-target/.env

```diff
@@ -0,0 +1,4 @@
+ETH_RPC_URL="https://ethereum.publicnode.com"
+FOUNDATION_SAFE="0x847B5c174615B1B7fDF770882256e2D3E95b9D92"
```

Comment:
We can remove `FOUNDATION_SAFE` as it's not an environment variable that's used in this execution.